### PR TITLE
add co-presidents, leadership, design lead to club structure

### LINKED ISSUE #140: document what presidents do

### PR DESCRIPTION
**Pull request by bobheadxi**
### Related Tickets

Closes #140 

### Changes

* add co-presidents, leadership, design lead to club structure
* add links to onboarding for more deets

### Checklist

* I've read the [Structuring Content](https://docs.ubclaunchpad.com/CONTRIBUTING#structuring-content) guide
* I've read up on our [Pull Request checks](https://docs.ubclaunchpad.com/CONTRIBUTING#checks)

### CODE REVIEW

File: handbook/manifesto.md [hunk 1]
```diff
@@ -47,21 +47,33 @@ As a club, Launch Pad supports every member’s experience through multiple ways
 
 ### Club Structure
 
+#### Co-Presidents
+
+Launch Pad is lead by two co-presidents. The role of the co-presidents is to act as the face of the club, manage our resources and accounts, and help the leadership team feel empowered and supported in initiatives they want to drive. The co-presidents do this by providing the leadership team with whatever guidance and resources you need, such as contacts, accounts, money, advice, and so on. Co-presidents also help ensure Launch Pad's initiatives and activities are aligned with the club's mission and objectives.
```
Comment (tarikeshaq) [inline handbook/manifesto.md:52]:
```suggestion
Launch Pad is lead by two co-presidents. The role of the co-presidents is to act as the face of the club, manage our resources and accounts, and help the leadership team feel empowered and supported in initiatives they want to drive. The co-presidents do this by providing the leadership team with whatever guidance and resources they need, such as contacts, accounts, money, advice, and so on. Co-presidents also help ensure Launch Pad's initiatives and activities are aligned with the club's mission and objectives.
```

File: handbook/manifesto.md [hunk 2]
```diff
@@ -47,21 +47,33 @@ As a club, Launch Pad supports every member’s experience through multiple ways
...
+
+Each year, one member is appointed as the **Design Lead**. The design lead's role is to support the design team and act as the bridge between the [leadership team](#leadership-team) and designers.
 
 #### Strategy Team
 
-UBC Launch Pad has a Strategy team that focusses on strategically growing the team. This team will focus on increasing campus awareness and forming strategic partnerships in order to further contribute to Launch Pad's vision to be the leading university technology club in Vancouver, Canada, or maybe even the world.
+UBC Launch Pad has a Strategy team that focusses on strategically growing the team. This team will focus on increasing campus awareness and forming strategic partnerships in order to further contribute to Launch Pad's vision to be the leading university technology club in Vancouver, Canada, or maybe even the world. Learn more about being on the strategy team in the [strategy onboarding guide](/handbook/onboarding/strategy.md).
```
Comment (tarikeshaq) [inline handbook/manifesto.md:76]:
```suggestion
UBC Launch Pad has a Strategy team that focuses on strategically growing the team. This team will focus on increasing campus awareness and forming strategic partnerships in order to further contribute to Launch Pad's vision to be the leading university technology club in Vancouver, Canada, or maybe even the world. Learn more about being on the strategy team in the [strategy onboarding guide](/handbook/onboarding/strategy.md).
```

File: handbook/manifesto.md (hunk 1 above)
Comment (bobheadxi) [reply handbook/manifesto.md:52]: @tarikeshaq is there a change in this comment? :o

Comment (tarikeshaq) [reply handbook/manifesto.md:52]: 😂 Sorry the block doesn't show it for some reason, but the "you need" -> "they need"

Comment (bobheadxi) [reply handbook/manifesto.md:52]: ah i see it now, thanks!

**Comment by bobheadxi** [comment]
> There's a section in the code of conduct: "Any of the following are reasons for expulsion from Launch Pad:"
> Has Launch Pad actually ever expelled anyone?

@johnagl AFAIK, never - most people just leave of their own accord :) This is another ancient line, that I suppose is basically a "we reserve the right to kick out people" clause

**Pull request merged**